.github/workflows/publish-aelf-samples-prod.yml

```diff
@@ -0,0 +1,60 @@
+on:
+  push:
+    tags:
+      - 'aelf-samples-v*'
```

Comment:
This file should not exists. Basically when someone puts up a template tag with version, it should also pack all aelf-samples contract into the contract template package.

Reply:
Removed prod-workflow